base/server/upgrade/10.10.0/03-RemoveLegacyTomcatJSSConfig.py

```diff
@@ -0,0 +1,43 @@
+# Authors:
+#     Alexander Scheel <ascheel@redhat.com>
+#
+# Copyright Red Hat, Inc.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+
+from __future__ import absolute_import
+import logging
+from lxml import etree
+
+import pki
+
+logger = logging.getLogger(__name__)
+
+
+class RemoveLegacyTomcatJSSConfig(pki.server.upgrade.PKIServerUpgradeScriptlet):
```

Comment:
@edewata -- do you want to review the upgrade script I added since you last reviewed? Then I can merge :-)

Reply:
It looks good, but do we need remove `ciphers.info` as well (e.g. from an existing PKI 10.9 instance)?

Reply:
Updated.

Reply:
You might want to add `self.backup(ciphers_info)`, but otherwise it looks good. Feel free to merge. Thanks!

Reply:
ACK, added. Thanks @edewata!